### ISSUE TITLE
Selling Items

### ISSUE THREAD

**Issue by Funyum** [WarEmu/WarBugs#19938]
I can't sell items to vendors by just right clicking on the item; I have to drag and drop the items onto the vendor sell list

![Screenshot (3)](https://user-images.githubusercontent.com/101758649/158709198-efbb3063-a8e4-4fec-810b-f19438278e94.png)
t

**Comment by Funyum** [comment]
I performed a fresh install, followed your guide line by line, double checked, and the launcher started and is still downloading 173,000+ files. I stopped it looked for a solution in forum and don't see one though I tried the one from the thread [[Turkish players, wait! Don't download that chunk of ~17000 files.] https://www.returnofreckoning.com/forum/viewtopic.php?f=4&t=42808); 3-Copy your "interface.myp" and paste it to the same folder and rename it as "ınterface.myp" (omit the double-quotes)(and yes we have "ı" character as the lowercase of "I" and this is what causes the problem); 4-Do the same with "audio.myp". Copy-paste and rename it as "audıo.myp".  
Nothing has worked, is there any reason I can't just download all files and press on?
![RoR 2022-03-18 ](https://user-images.githubusercontent.com/101758649/159017296-d3988e70-6ae9-4d3c-b4d8-533f20baf835.png)
